### ISSUE TITLE
Issue with "<pre>: The Preformatted Text element": (short summary here please)

### ISSUE THREAD

**Issue by jcamiel** [mdn/content#9149]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre

#### What information was incorrect, unhelpful, or incomplete?

I'm looking for information whether I have to escape certain characters inside a pre tag or not.
One of the samples in the page is:

```
<figure role="img" aria-labelledby="cow-caption">
  <pre>
  ___________________________
< I'm an expert in my field. >
  ---------------------------
         \   ^__^
          \  (oo)\_______
             (__)\       )\/\
                 ||----w |
                 ||     ||
  </pre>
  <figcaption id="cow-caption">
    A cow saying, "I'm an expert in my field." The cow is illustrated using preformatted text characters.
  </figcaption>
</figure>
```

The `<` in `< I'm an expert in my field. >` seems not to be escaped, but I've the impression that every `<`, `>`, `&`, `'`, `"` should be escaped inside the `<pre>` tag.

#### Specific section or headline?

The sample sections <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre#example> should maybe add a sample with a proper escaping.

#### What did you expect to see?

An statement about escaping inside `<pre>` tag.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/html/element/pre`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/pre/index.html
* Last commit: https://github.com/mdn/content/commit/055ebe07415a761004e00673de18f67570e6ab85
* Document last modified: 2021-06-09T04:39:20.000Z

</details>

**Comment by digi-booster** [comment]
see behind the code - https://github.com/mdn/content/blob/main/files/en-us/web/html/element/pre/index.html#:~:text=%3Cpre%3E%26lt%3Bfigure,%3C/pre%3E

**Comment by jcamiel** [comment]
> see behind the code - https://github.com/mdn/content/blob/main/files/en-us/web/html/element/pre/index.html#L91

In my comprehension, the example should be what you've to write in plain text.
If you write this sample in vi for instance, you should write exactly:

```html

**Comment by digi-booster** [comment]
Yes, the HTML part should be added separately on the page like in the above example.
